A fixed molecular charge for the molecule generation. (#73)
grimme-lab/MindlessGen: CHANGELOG.md

- function which is able to printout the xyz coordinates to the terminal similar to the `.xyz` layout
- elements 87 to 103 are accessible via the element composition. If `xtb` is the engine, the elements will be replaced by their lighter homologues.
- support for `python-3.13`

> - function which is abel to set a molecular charge and ensure `uhf = 0`

Reviewer: ```suggestion
- option to set a fixed molecular charge, while ensuring `uhf = 0`
```
